### ISSUE TITLE
Improper bytes/unicode in glances_hddtemp.py

### ISSUE THREAD

**Issue by stephane-martin** [nicolargo/glances#887]
Hello,

with python3.4 and hddtemp enabled, i can't get access to the web interface.

Digging in the HTTP traffic shows an API call to api/2/all that fails, with a JSON encoding error (choking on bytes data).

Indeed `glances_hddtemp.py` seems to have an encoding problem with the temperature value.

In `GlancesGrabHDDTemp.__update__`:

`temperature = fields[offset + 3]` means that temperature has 'bytes' type

then `hddtemp_current['value'] = float(temperature) if isinstance(temperature, numbers.Number) else temperature` ==> `if isinstance(temperature, numbers.Number)` is always False ==> `hddtemp_current['value']` has also 'bytes' type

instead, maybe do:

```python
try:
    hddtemp_current['value'] = float(temperature)
except ValueError:
    hddtemp_current['value'] = nativestr(temperature)
```
Hope it helps :)

**Comment by nicolargo** [comment]
@notFloran / @nclsHart : Can you have a llok on this encoding issue in the Web UI ?

**Comment by stephane-martin** [comment]
ping ?

**Comment by nicolargo** [comment]
@stephane-martin What is **nativestr** ?

**Comment by stephane-martin** [comment]
a function defined in `glances/compat.py`, i didn't invent it :)

**Comment by nicolargo** [comment]
Corrected in the latest HEAD DEVELOp version (will be included in the 2.7 release)
